src/main/java/org/dcsa/bkg/model/transferobjects/ShipmentCutOffTimeTO.java

```diff
@@ -0,0 +1,19 @@
+package org.dcsa.bkg.model.transferobjects;
+
+import com.fasterxml.jackson.annotation.JsonFormat;
+import lombok.Data;
+import org.dcsa.bkg.model.enums.CutOffDateTimeCode;
+
+import javax.validation.constraints.NotNull;
+import java.time.OffsetDateTime;
+
+@Data
+public class ShipmentCutOffTimeTO {
+
+  @NotNull(message = "Cut Off Date Time Code is required.")
+  private CutOffDateTimeCode cutOffDateTimeCode;
+
+  @NotNull(message = "Cut Off Date Time is required.")
+  @JsonFormat(pattern = "yyyy-MM-dd")
```

Comment:
Let's use only `@JsonFormat(shape = JsonFormat.Shape.STRING)` for now. DCSA-Core might have problems with this.
Plus swagger says 
```
cutOffDateTimestring($date-time)
example: 2019-11-12T07:41:00+08:30
```
So it shouldn't be `yyyy-MM-dd` here anyways.